src/components/templates/Audience/Audience.tsx

```diff
@@ -12,6 +12,12 @@ import { addReaction } from "store/actions/Reactions";
 
 import { makeUpdateUserGridLocation } from "api/profile";
 
+import { useDispatch } from "hooks/useDispatch";
+import { useRecentVenueUsers } from "hooks/users";
+import { useSelector } from "hooks/useSelector";
+import { useUser } from "hooks/useUser";
+import { useVenueId } from "hooks/useVenueId";
+
```

Comment:
Why were these moved from where they were? `hooks/*` live below `utils/*` in the import order groupings.